atoma-state/src/handlers.rs

```diff
@@ -607,6 +607,18 @@ pub(crate) async fn handle_stack_attestation_dispute_event(
     Ok(())
 }
 
+pub(crate) async fn handle_node_registration_event(
+    state_manager: &AtomaStateManager,
+    event: NodeRegisteredEvent,
+    address: String,
+) -> Result<()> {
+    state_manager
+        .state
+        .insert_new_node(event.node_small_id.inner as i64, address)
+        .await?;
```

Comment:
log potential error here

Reply:
the errors are propagated all the way to error `handle_tasks_results` in `main.rs` We should not double log errors.